src/certificateList.jsx

```diff
@@ -0,0 +1,235 @@
+
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2020 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// import cockpit from "cockpit";
+import React from "react";
+import moment from "moment";
+
+import {
+    Accordion,
+    AccordionItem,
+    AccordionContent,
+    AccordionToggle,
+    Flex,
+    FlexItem,
+    FlexModifiers
+} from "@patternfly/react-core";
+
+import './certificateList.css';
+import '../lib/form-layout.less';
+import { getRequests, getRequest, getCA } from "./dbus.js";
+
+const _ = cockpit.gettext;
+function prettyTime(unixTime) {
+    moment.locale(cockpit.language, {
+        longDateFormat : {
+            LT: "hh:mm:ss",
+            L: "DD/MM/YYYY",
+        }
+    });
+    const yesterday = _("Yesterday");
+    const today = _("Today");
+    moment.locale(cockpit.language, {
+        calendar : {
+            lastDay : `[${yesterday}] LT`,
+            sameDay : `[${today}] LT`,
+            sameElse : "L LT"
+        }
+    });
+
+    return moment(Number(unixTime) * 1000).calendar();
+}
+
+class CertificateList extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            certs: [],
+            cas: {},
+            expanded: [],
+        };
+        this.toggle = this.toggle.bind(this);
+        this.onValueChanged = this.onValueChanged.bind(this);
+
+        getRequests()
+                .fail(error => {
```

Comment:
fail is jquery thing. Can you please use `.catch`? And also it should be the last item in the chain and not the first one.